cmd/crictl/sandbox.go

```diff
@@ -322,7 +374,7 @@ var listPodCommand = &cli.Command{
 		if err != nil {
 			return err
 		}
-		if err = ListPodSandboxes(runtimeClient, opts); err != nil {
+		if err = OutputPodSandboxes(runtimeClient, opts); err != nil {
```

Comment:
if filtered by reference, what image reference will the output use as a main one? My understanding is that even with the filter, the output may show  different reference as a "main" name of an image and tag one filtered by in a list of alternative names. Is it right? I wonder this is actually the case and whether we can either improve output or make it clear in docs so not to confuse users

Reply:
Hm, pods are not filterable by reference, are we speaking about `crictl inspecti`? My main goal was to mimic closely as possible the existing `crictl` `pods`, `ps` and `images` subcommand flags to avoid confusion.